src/public.js

```diff
@@ -58,6 +69,13 @@ async function doQueryIssues(owner, repo, labels, state, creator) {
        * You can identify pull requests by the pull_request key.
        */
       if (a && b && iss.pull_request === undefined) {
+        if (excludeLabels) {
+          const labels = dealStringToArr(excludeLabels);
```

Comment:
The result of `dealStringToArr(excludeLabels)` could be extracted to outter scope.

Reply:
PR welcome ~